pywbem/cim_operations.py

```diff
@@ -54,6 +54,30 @@
 :meth:`~pywbem.WBEMConnection.InvokeMethod`             Invoke a method on a target instance or on a target class
 ------------------------------------------------------  --------------------------------------------------------------
 :meth:`~pywbem.WBEMConnection.ExecQuery`                Execute a query in a namespace
+------------------------------------------------------- ---------------------------------------------------------
+:meth:`~pywbem.WBEMConnection.OpenEnumerateInstances`   Open an enumeration session to retrieve instances of
+                                                        of a class (including instances of its subclass)
+:meth:`~pywbem.WBEMConnection.OpenAssocatorInstances`   Open an enumeration session to retrieve the instances
+                                                        associated to a source instance
+:meth:`~pywbem.WBEMConnection.OpenReferenceInstances`   Open an enumeration session to retrieve the instances
+                                                        that reference a source instance
+:meth:`~pywbem.WBEMConnection.PullInstancesWithPath`    Continue enumeration of an enumeration session opened
+                                                        with OpenEnumerateInstances, OpenAssociators, or
+                                                        OpenReferences
+:meth:`~pywbem.WBEMConnection.OpenEnumerateInstancePaths`   Open an enumeration session to retrieve instances of
+                                                        of a class (including instances of its subclass)
+:meth:`~pywbem.WBEMConnection.OpenAssocatorInstancePaths`  Open an enumeration session to retrieve the instances
+                                                        associated to a source instance
+:meth:`~pywbem.WBEMConnection.OpenReferenceInstancesPaths`  Open an enumeration session to retrieve the instances
+                                                        that reference a source instance
+:meth:`~pywbem.WBEMConnection.PullInstance Paths`       Continue enumeration of an enumeration session opened
+                                                        with OpenEnumerateInstancePaths, OpenAssociatorInstancePaths,
+                                                        or OpenReferenceInstancePaths
+:meth:`~pywbem.WBEMConnection.OpenExecQuery`            Open query request to retrieve instances defined by
+                                                        the query parameter in a namespace
+:meth:`~pywbem.WBEMConnection.PullInstances`            Continue enumeration of an enumeration session opened
+                                                        with OpenExecQuery
+:meth:`~pywbem.WBEMConnection.CloseEnumeration`         Close an enumeration session in process.
```

Comment:
I suggest adding a note at the bottom pointing out that EnumerationCount is not supported.